data-structures-queues/is-empty.js

```diff
@@ -0,0 +1,8 @@
+/* exported isEmpty */
+
+function isEmpty(queue) {
+  const NewQ = queue;
+  if (NewQ.peek() === undefined) {
```

Comment:
No need to create and extra `const` here, you can just use `queue.peek()` in your `if` statement